browser/ui/views/profiles/brave_avatar_toolbar_button.cc

```diff
@@ -64,14 +64,18 @@ void BraveAvatarToolbarButton::SetHighlight(
   std::u16string revised_highlight_text;
   if (browser_->profile()->IsTor()) {
     revised_highlight_text =
-        l10n_util::GetStringUTF16(IDS_TOR_AVATAR_BUTTON_LABEL);
+        l10n_util::GetStringUTF16(IDS_LOCATION_BAR_OPEN_IN_TOR);
```

Comment:
Let's add new label for this.
Although new label will have same string with `IDS_LOCATION_BAR_OPEN_IN_TOR`,
it could prevent regression when location bar label is changed.